modules/recommended-etcd-practices.adoc

```diff
@@ -30,13 +26,10 @@ The following hard disk features provide optimal etcd performance:
 
 Avoid NAS or SAN setups, and spinning drives. Always benchmark using utilities such as `fio`. Continuously monitor the cluster performance as it increases.
```

Comment:
For clarity add "by" before "using". No formatting is needed for the term "fio".

Suggested revision: `Avoid NAS or SAN setups and spinning drives. Always benchmark by using utilities such as fio. Continuously monitor the cluster performance as it increases.`